openstf/model/objective.py

```diff
@@ -164,10 +164,10 @@ def get_params(self, trial: optuna.trial.FrozenTrial) -> dict:
         """get parameters for XGB Regressor Objective
         with objective specific parameters.
 
-                                Args: trial
+            Args: trial
 
-                                Returns:
-                                    dict: {parameter: hyperparameter_value}
+            Returns:
+                dict: {parameter: hyperparameter_value}
         """
         default_params = super().get_default_params(trial)
```

Comment:
It looks like this part is identical for all models. Is it possible to move it to the parent-class?